### ISSUE TITLE
Use perform_now instead of perform

### ISSUE THREAD

**Issue by NeilBetham** [plashchynski/crono#58]
Would it be possible to use ActiveJob's `peform_now` instead of `perform`? The main reason I ask is that using `peform_now` will call all of the ActiveJob callbacks and error handlers. The error handler callback is particularly important for reporting problems to a error reporting system like Sentry. Is there a reason `perform` was used instead of `perform_now`?

**Comment by MohamedBassem** [comment]
Yes please. I spent 2 hours trying to find why job failures doesn't trigger airbrake notifications until I found that the `rescue_from` of the ActiveJob is not triggered because `Job.new.perform` is used instead of `perform_now`.

**Comment by edwardmp** [comment]
I agree. Is there any reason perform is used because of perform_now?

**Comment by edwardmp** [comment]
As a workaround you can do something like this:

```
class NewRegistrationProcessor
  def perform
    NewRegistrationProcessorJob.perform_now
  end
end

Crono.perform(NewRegistrationProcessor).every 1.minute
```

But I agree it would be better if perform_now is called by default

**Comment by sina-s** [comment]
I have the exact same use-case, reporting errors to Sentry. I came up with another work around that doesn't need monkey patching Crono. Assuming your jobs are subclass of `ApplicationJob`, define your `perform` inside `ApplicationJob` and change the `perform` method name to something else (in my case `perform!`) in your job class. Not sure how graceful this work around is, as it requires to change all of your job classes.
```
class ApplicationJob < ActiveJob::Base
  def perform
    perform!
  rescue StandardError => e
    Raven.capture_exception(e)
  end 
end
```
```
class MyFrequntTask < ApplicationJob
  def perform!
     ...
   end
end
```
